src/safecast.yaml

```diff
@@ -0,0 +1,31 @@
+---
+description: >
+  Safecast is a global volunter-centered **Citizen Science** project working to
+  empower people with open data about their environments.
+license: MIT
+name: Safecast
+programmingLanguages:
+  - C++
+  - C
+  - JavaScript
+  - PHP
+  - Go
+rating: 5
+repoUrl: "https://github.com/safecast"
+sdgs: [3, 9]
+websiteUrl: "https://blog.safecast.org/"
+
+contributionGuidelinesUrl: "https://blog.safecast.org/volunteer/"
+logoUrl: "https://i1.wp.com/blog.safecast.org/wp-content/uploads/2011/07/\
+         safecast-square-ct.png"
```

Comment:
Are we allowed to use this Logo?
Because on their license page, it says:

> The term “Safecast” and the safecast logo are Registered Trademarks of the Momoko Ito Foundation, a 501(c)3 Non-profit
> Usage must be approved by us.

If it's approved, maybe we should archive that somewhere?

Reply:
@AblingerOscar do we need to be so serious regarding license usage for logos?
* Logo licenses exist to prevent unwanted usage like reusing the brand image for a different purpose.
* We are a FOSS project, not for profit.
* All we do is listing their project, for their own benefit.

We're basically only doing advertisement for these groups. I'd love to see a project come and complain because we listed them with their logo for people to come and help them :rofl:

Reply:
I know, I know ^^
It's probably stupid, yes. Just randomly stumbled across it